pkg/constants/constants.go

```diff
@@ -0,0 +1,8 @@
+package constants
+
+const (
+	UPSImage        = "quay.io/aerogear/unifiedpush-configurable-container:2.4"
```

Comment:
```suggestion
	UPSImage        = "quay.io/aerogear/unifiedpush-configurable-container:2.4.0-1"
```

The `2.4` floating tag was there for image streams. We should now be using the most specific tag that we can. If we need to update to a new image, we update the operator.

I know that the other images are floating tags for the moment also, we can address that separately in the future, when we start using digests rather than tags.

Reply:
Yeah, let's do it later. I want to merge this 2 week PR